public/locations-list-map.js

```diff
@@ -276,94 +304,69 @@ $(function () {
 
     showList && createContent(window.data_by_location);
 
-    const stateFilter = {};
-    let hasStateFilter = false;
+    const filters = createFilters();
+
+    // Update filters to match any ?state= params
+    const stateParams = searchParams.getAll('state').map(state => state.toUpperCase());
+    const states = stateParams.map(param => param.split(',')).reduce((acc, val) => acc.concat(val), []);
```

Comment:
nit: I wonder if this would be faster with a spread operator and push rather than concat?